repo/packages/D/datastax-dse/400/config.json

```diff
@@ -0,0 +1,2196 @@
+          "type": "integer",
+          "default": 2
+        },
+        "graph_gremlin_server_maxContentLength": {
+          "type": "integer",
+          "default": 65536000
+        },
+        "graph_gremlin_server_maxChunkSize": {
+          "type": "integer",
+          "default": 4096000
+        },
+        "graph_gremlin_server_maxInitialLineLength": {
+          "type": "integer",
+          "default": 4096
+        },
+        "graph_gremlin_server_maxHeaderSize": {
+          "type": "integer",
+          "default": 8192
+        },
+        "graph_gremlin_server_maxAccumulationBufferComponents": {
+          "type": "integer",
+          "default": 1024
+        },
+        "graph_gremlin_server_resultIterationBatchSize": {
+          "type": "integer",
+          "default": 64
+        },
+        "graph_gremlin_server_useEpollEventLoop": {
+          "type": "boolean",
+          "default": false
+        },
+        "graph_gremlin_server_gremlinPool": {
+          "type": "integer",
+          "default": 8
+        },
+        "dsefs_enabled": {
+          "description": "Enable/Disable DSE File System",
+          "type": "boolean",
+          "default": true
+        },
+        "dsefs_data_disk_type": {
+          "description": "DSE FS persistent disk type [ROOT, MOUNT]",
+          "type": "string",
+          "enum": [
+            "ROOT",
+            "MOUNT"
+          ],
+          "default": "ROOT"
+        },
+        "dsefs_disk": {
+          "description": "DSE File System size in MB",
+          "type": "integer",
+          "default": 10
+        },
+        "dsefs_keyspace_name": {
+          "type": "string",
+          "default": "dsefs"
+        },
+        "dsefs_public_port": {
+          "type": "integer",
+          "default": 5598
+        },
+        "dsefs_private_port": {
+          "type": "integer",
+          "default": 5599
+        },
+        "dsefs_data_directories_storage_weight": {
+          "type": "number",
+          "default": 1.0
+        },
+        "dsefs_data_directories_min_free_space": {
+          "type": "string",
+          "default": "5368709120"
+        },
+        "dsefs_service_startup_timeout_ms": {
+          "type": "string",
+          "default": "30000"
+        },
+        "dsefs_service_close_timeout_ms": {
+          "type": "string",
+          "default": "600000"
+        },
+        "dsefs_server_close_timeout_ms": {
+          "type": "string",
+          "default": "2147483647"
+        },
+        "dsefs_compression_frame_max_size": {
+          "type": "string",
+          "default": "1048576"
+        },
+        "dsefs_query_cache_size": {
+          "type": "string",
+          "default": "2048"
+        },
+        "dsefs_query_cache_expire_after_ms": {
+          "type": "string",
+          "default": "2000"
+        },
+        "dsefs_gossip_round_delay_ms": {
+          "type": "string",
+          "default": "2000"
```

Comment:
I noticed that the default value for `dsefs_gossip_round_delay_ms` changed from `5000` to `2000`, was that intentional?

```diff
@@ -1675,15 +1698,15 @@
         },
         "dsefs_gossip_round_delay_ms": {
           "type": "string",
+          "default": "2000"
+        },
+        "dsefs_gossip_startup_delay_ms": {
+          "type": "string",
           "default": "5000"
         },
```

Reply:
From the PR here: https://github.com/mesosphere/dse-private/commit/56977364d29ec832bca255cfc9d67e7c72c5e24a#diff-142c2b6baf1e36814031ea6e37da9077R1689 I would say that this was intentional.

Reply:
yes, it's intentional.